Python/CMakeLists.txt

```diff
@@ -103,6 +103,8 @@ set(PRP_AUDIO_SOURCES
     PRP/Audio/py2WayWinAudible.cpp
     PRP/Audio/pyAudible.cpp
     PRP/Audio/pyAudibleNull.cpp
+    PRP/Audio/pyEAXEffects.cpp
+    PRP/Audio/pyEAXListenerMod.cpp
```

Comment:
The file is misnamed as `pyEaxListenerMod.cpp`, which will error on case sensitive file systems, such as Linux and macOS.

Reply:
Yep, Git didn't catch it because windoz. Thanks for the heads up.